text/src/autogluon/text/automm/predictor.py

```diff
@@ -551,11 +561,23 @@ def _fit(
             fast_dev_run=config.env.fast_dev_run,
             val_check_interval=config.optimization.val_check_interval,
         )
-        trainer.fit(
-            task,
-            datamodule=train_dm,
-            ckpt_path=self._ckpt_path,  # this is to resume training that was broken accidentally
-        )
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                ".*does not have many workers which may be a bottleneck. "
+                "Consider increasing the value of the `num_workers` argument` "
+                ".* in the `DataLoader` init to improve performance.*"
+            )
+            warnings.filterwarnings(
+                "ignore",
+                "Checkpoint directory .* exists and is not empty."
+            )
```

Comment:
Consider to later warp this part + log inside a context manager: https://docs.python.org/3/library/contextlib.html

Reply:
Done.